### DISCUSSION TITLE
Fatal errors for FBA inventory reports

### DISCUSSION THREAD

**Discussion by Sandeep0393** [amzn/selling-partner-api-models#3340] [General] [answered]
We are getting fatal error for FBA inventory reports fetched via Report API. What could be the reason for FATAL errors? Also, is there any changes to rate limits recently for reports API?

**Comment by nirmay** [comment, marked as answer]
Some of these reports (including inventory) cant be called multiple times
every 30 minutes, so you will need to wait until you call it again.
the specifics are not very clear on the docs.

On Fri, Jun 10, 2022 at 12:52 PM Sandeep Chittaluri <
***@***.***> wrote:

> We are getting fatal error for FBA inventory reports fetched via Report
> API. What could be the reason for FATAL errors? Also, is there any changes
> to rate limits recently for reports API?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/amzn/selling-partner-api-docs/discussions/2340>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEIF65YWRJS4ZUZYBSZV3ILVOLUNDANCNFSM5YMWQJDA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Mayuresh A Nirhali
